setup.py

```diff
@@ -77,6 +77,7 @@
             "pep8-naming==0.12.1",
             "pylint>=2.9.3,<=2.13.8",
             "types-setuptools>=57.4.7,<=57.4.14",
+            "wheel>=0.37.1",
```

Comment:
I think we should either pin to
```python
"wheel==0.37.1"
```
or constrain the next-up version to avoid breaking changes:
```python
"wheel>=0.37.1,<1.0.0"
```

Reply:
Sure, I will pin it to `wheel==0.37.1`.